package.json

```diff
@@ -34,5 +34,20 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "big.js": "^6.0.3",
+    "eslint": "^7.11.0",
+    "eslint-config-airbnb": "^18.1.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.20.6",
+    "eslint-plugin-react-hooks": "^2.5.1",
+    "prop-types": "^15.7.2",
```

Comment:
`prop-types should also be added as a regular dependency. You can use the command
```sh
npm install prop-types --save
```
to have it installed as a regular dependecy